Description: Change location of hfi1 driver in libibverbs config
 The hfi1 libibverbs driver is installed under /usr/lib/libibverbs. That location
 is consitent with the one used by the mlx4 driver on Debian. This patch corrects
 the location in the libibverbs driver config.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: not-needed
Last-Update: <2018-05-16>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/hfi1.driver
+++ b/hfi1.driver
@@ -1 +1 @@
-driver hfi1verbs
+driver /usr/lib/libibverbs/libhfi1verbs
